pex/pex.py

```diff
@@ -94,6 +94,18 @@ def _extras_paths(cls):
     extras_paths = filter(None, makefile.get('EXTRASPATH', '').split(':'))
     for path in extras_paths:
       yield os.path.join(standard_lib, path)
+    # macOS Sierra support
+    framework = sysconfig.get_config_var("PYTHONFRAMEWORK")
+    if not framework:
+      raise StopIteration()
```

Comment:
better to simply `return` in these cases than to manually raise `StopIteration`:

```
if not framework:
  return
```